content/developer/tutorials/discover_js_framework/01_components.rst

```diff
@@ -287,6 +287,10 @@ The final touch is to let the user delete a todo.
 
 Owl has a powerful `slot <{OWL_PATH}/doc/reference/slots.md>`_ system to allow you to write generic
 components. This is useful to factorize the common layout between different parts of the interface.
+When you create a generic component, you should like to insert information like title or description in your component but obviously, this informations will change from a component to another. Let's take the example of few cards to present few films, each card has a different title and a different description. 
```

Comment:
i don't think your explanation is clear (communication is hard!). Here is a suggestion, adapted from your text:
```

Imagine that you want to display some cards with each the title and a description of a film.
Clearly, the parent component needs to *define* a content for each card, and each card need
to *insert* the content given by the parent component. This is done by using the directives
`t-set-slot="slotname"` (to define) and `t-slot="slotname"` (to insert).
```

Feel free to take this and improve it if you can

Reply:
also, note that the text above is written with shorter column width, which is useful to read the content in an editor. Try to use the same formatting